src/Player.js

```diff
@@ -6,11 +10,29 @@ class Player {
 
   solvePuzzle() {
 
+  }
+  
+  getPlayerScore() {
+
   }
 
   spinWheel() {
     
   }
+
+  buyVowel() {
+    if(this.score > 100) {
+      //buy vowel becomes enabled
+      //dom updates to show enabled
+      //when enabled, player can enter in 1 vowel and if it's in the letter if will show up on the puzzle bank
```

Comment:
I'm sure you meant to add it into the pseudo code, but...
REMINDER: subtract 100 points from the round total of that current player